React/Lesson41/redux-app/src/Cart/Cart.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { incrementClickCount } from './redux/actions';
+
+export const CartContext = React.createContext();
```

Comment:
as I said on lecture
Redux should be Single Source of True
so you can't use Context and Redux in one app